client/assets/net.veloren.airshipper.metainfo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Adam Blanchet-->
+<component type="desktop-application">
+  <id>net.veloren.airshipper</id>
+  
+  <name>Airshipper</name>
+  <summary>Airshipper is the official launcher for Veloren.</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <content_rating type="oars-1.0">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="language-profanity">mild</content_attribute>
+    <content_attribute id="language-humor">mild</content_attribute>
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release version="0.4.2" date="2020-12-16"/>
```

Comment:
There was a similar issue with nxengine-evo's AppStream file which was (at least for the snap package) mitigated by adding an [XSLT transformation](https://github.com/nxengine/nxengine-evo/blob/master/platform/xdg/appdata-add-version.xslt) to automatically set the correct version/git hash for nightly builds. Maybe something similar could be done here as well to extract the current version from git?

Very happy to see this PR as I'm looking into shipping Airshipper in the veloren snap :+1:

Reply:
Yeah 0.5 will probably be the biggest release ever with a lot of useful things. Would be glad if someone figures out how to automate that based on this XSLT thing but if not just documenting it in the WORKFLOW.md should be good enough.

Reply:
I've added a section for this (and screenshots) in WORKFLOW.md. Let me know if you'd prefer to have it include more details for example for the format of description and such (since for the current release I haven't put a description).